docs/index.md

```diff
@@ -27,16 +27,16 @@ If you want more accurate results or Cloud Costs from your provider's bill pleas
 
 The [OpenCost Specification](specification) describes a vendor-neutral implementation for Kubernetes cost monitoring.
 
+## Using the UI
+
+The OpenCost UI provides a visualization of the Kubernetes allocations and the related Cloud Costs. This walkthrough highlights some of the components of the OpenCost UI:
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/j_4aaE5zk7M?si=N01C23OyAx3qa12b" title="OpenCost UI walkthrough" frameborder="1" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```

Comment:
this width is much smaller than main context width. can we make wider?

Reply:
I made it 150% larger